lib/Wikia/src/Vignette/UrlGenerator.php

```diff
@@ -31,6 +31,10 @@ class UrlGenerator {
 
 	const REVISION_LATEST = 'latest';
 
+	const BASE_URL_PROD = 'http://vignette<SHARD>.wikia.nocookie.net';
```

Comment:
We've kept integration with Wikia/Mediawiki to `includes/wikia/vignette` and `lib/Wikia/src/Vignette` as an environment-agnostic library for generating image URLs, so I'd say that `includes/wikia/vignette` is probably a better place for these configurations to live, and even better would be somewhere in the config repo if that's easy to do.